### ISSUE TITLE
Scyllatop crashes when trying to scroll down

### ISSUE THREAD

**Issue by nirmaayan** [scylladb/scylla#1396]
Scylla version (or git commit hash): 1.2
Cluster size: 3 
OS (RHEL/CentOS/Ubuntu/AWS AMI): centos 7.2

running scyllatop command 
$ scyllatop
Trying to scroll - result scyllatop crashes with:

```
Traceback (most recent call last):
  File "/usr/lib/scylla/scyllatop/scyllatop.py", line 98, in <module>
    main(arguments.metricPattern, arguments.interval, collectd)
  File "/usr/lib/scylla/scyllatop/scyllatop.py", line 44, in main
    loop.run()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 271, in run
    self.screen.run_wrapper(self._run)
  File "/usr/lib64/python2.7/site-packages/urwid/raw_display.py", line 241, in run_wrapper
    return fn()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 336, in _run
    self.event_loop.run()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 707, in run
    self._loop()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 786, in _loop
    self._watch_files[fd]()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 387, in _update
    self.process_input(keys)
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 493, in process_input
    something_handled |= bool(self.unhandled_input(k))
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 539, in unhandled_input
    return self._unhandled_input(input)
  File "/usr/lib/scylla/scyllatop/userinput.py", line 18, in __call__
    if keypress.upper() not in self._viewMap:
AttributeError: 'tuple' object has no attribute 'upper'


```

**Comment by nirmaayan** [comment]
Traceback (most recent call last):
  File "/usr/lib/scylla/scyllatop/scyllatop.py", line 98, in <module>
    main(arguments.metricPattern, arguments.interval, collectd)
  File "/usr/lib/scylla/scyllatop/scyllatop.py", line 44, in main
    loop.run()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 271, in run
    self.screen.run_wrapper(self._run)
  File "/usr/lib64/python2.7/site-packages/urwid/raw_display.py", line 241, in run_wrapper
    return fn()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 336, in _run
    self.event_loop.run()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 707, in run
    self._loop()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 786, in _loop
    self._watch_files[fd]()
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 387, in _update
    self.process_input(keys)
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 493, in process_input
    something_handled |= bool(self.unhandled_input(k))
  File "/usr/lib64/python2.7/site-packages/urwid/main_loop.py", line 539, in unhandled_input
    return self._unhandled_input(input)
  File "/usr/lib/scylla/scyllatop/userinput.py", line 18, in __call__
    if keypress.upper() not in self._viewMap:
AttributeError: 'tuple' object has no attribute 'upper'

scyllatop
[scyllatop_output.txt](https://github.com/scylladb/scylla/files/341172/scyllatop_output.txt)

**Comment by penberg** [comment]
@tzach Why did you reopen the issue?

**Comment by tzach** [comment]
Allowing @nirmaayan and me to test it

**Comment by penberg** [comment]
@tzach OK. Please include a comment when you reopen an issue.

**Comment by tzach** [comment]
@penberg I will.

**Comment by nirmaayan** [comment]
@slivne @tzach 
Closed in 1.3
